app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragment.kt

```diff
@@ -35,8 +37,16 @@ class CompletedStoryListFragment : InjectableFragment() {
     container: ViewGroup?,
     savedInstanceState: Bundle?
   ): View? {
-    val args = checkNotNull(arguments) { "Expected arguments to be passed to CompletedStoryListFragment" }
-    val internalProfileId = args.getInt(COMPLETED_STORY_LIST_FRAGMENT_PROFILE_ID_KEY, -1)
-    return completedStoryListFragmentPresenter.handleCreateView(inflater, container, internalProfileId)
+    val args =
+      checkNotNull(arguments) { "Expected arguments to be passed to CompletedStoryListFragment" }
+    val internalProfileId = args
+      .getInt(
+        COMPLETED_STORY_LIST_FRAGMENT_PROFILE_ID_KEY, -1
+      )
```

Comment:
Nit changes for better code readability:
```
    val args = checkNotNull(arguments) {
      "Expected arguments to be passed to CompletedStoryListFragment"
    }
    val internalProfileId = args.getInt(
      COMPLETED_STORY_LIST_FRAGMENT_PROFILE_ID_KEY, -1
    )
```

Reply:
ok, Thanks for review.